H2MLauncher.UI/Model/GlobalKeyboardHook.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Input;
+
+namespace H2MLauncher.UI.Model
+{
+    public class GlobalKeyboardHook : IDisposable
```

Comment:
make it sealed, then the IDispose.Dispose won't complain as it doesn't need to account for inheritance disposals